ui/app/TokenList.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "Chaosnet supported tokens",
+  "tokens": [
+    {
+      "chainId": 10,
+      "address": "0x7F5c764cBc14f9669B88837ca1490cCa17c31607",
+      "name": "USDCoin",
+      "decimals": 6,
+      "symbol": "USDC.e"
+    },
+    {
+      "chainId": 58008,
```

whoops this is Arbitrum's chain ID
```suggestion
      "chainId": 42161,
```